superset/security/manager.py

```diff
@@ -57,15 +56,15 @@
 logger = logging.getLogger(__name__)
 
 
-class SupersetSecurityListWidget(ListWidget):
+class SupersetSecurityListWidget(ListWidget):  # pylint: disable=too-few-public-methods
```

Comment:
should we deny-list this one `too-few-public-methods`?

Reply:
I'll disable it for the file.